config-templates/authmemcookie.php

```diff
@@ -81,6 +81,33 @@
 	 */
 	'memcache.port' => 11211,
 
+	/*
+	 * This option specifies how the attributes are represented in the memcache store.
+	 * Up until SimpleSamlPhp 1.13, the format was "implode".
```

Comment:
s/SimpleSamlPhp/SimpleSAMLphp/

Just trying to keep coherence with the format we chose to use for the name :smiley: